spec/forms/register_nonprofit_form/nonprofit_form_spec.rb

```diff
@@ -1,80 +1,77 @@
 # License: AGPL-3.0-or-later WITH Web-Template-Output-Additional-Permission-3.0-or-later
 require "rails_helper"
 
-RSpec.describe RegisterNonprofitForm::NonprofitForm, :type => :model do
-
+RSpec.describe RegisterNonprofitForm::NonprofitForm, type: :model do
   describe "#website" do
     it { is_expected.to allow_value("www.exa.cs").for(:website) }
-    it "normalizes with http" do
 
+    it "normalizes with http" do
       form = described_class.new(website: "www.exa.cs")
 
       expect(form.website).to eq "http://www.exa.cs"
     end
-
   end
 
   it "validates email" do
     form = described_class.new({
-              email: "noemeila",
-              phone: "notphone",
-              website: ""
-          })
+      email: "noemeila",
+      phone: "notphone",
+      website: ""
+    })
     form.valid?
 
     # we should have the email validation from User model
     expect(form.errors.messages_for(:email)).to be_one
   end
 
-  context 'slug management' do
+  describe "slug management" do
     let(:nonprofit_attributes) do
       attributes_for(
         :nonprofit_base,
         slug: nil,
         zip_code: 51455
       ).slice(
-          :name,
-          :city,
-          :email,
-          :state_code,
-          :phone,
-          :website,
-          :zip_code
-        )
-        
+        :name,
+        :city,
+        :email,
+        :state_code,
+        :phone,
+        :website,
+        :zip_code
+      )
     end
 
-    it 'creates slug when easy' do
+    it "creates slug when easy" do
       form = described_class.new(nonprofit_attributes) # nil so it autocreates the slug
-      
-      expect(form.valid?).to eq true
-      
+
+      expect(form.valid?).to be true
```

Comment:
`be` and `eq` aren't exactly equivalent. As mentioned  below though, this would be easier if written as `expect(form).to be_valid`